### PR TITLE
EndTime is not displayed when SetMediaClock set (UpdateMode=COUNTDOWN) with valid startTime and EndTime

### PR DESCRIPTION
**Pull request by ValeriiMalkov**
Implements/Fixes [#627](https://github.com/smartdevicelink/sdl_hmi/issues/627)

This PR is **ready** for review.

### Testing Plan
Manual testing

### Summary
Added countdown timer displaying

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by ValeriiMalkov** [comment]
Hi @theresalech ! This PR is ready for Livio team review!

File: app/model/sdl/MediaModel.js [hunk 1]
```diff
@@ -226,20 +226,35 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
...
+          min = parseInt(position / 60) % 60, // minutes
+          sec = position % 60; // seconds
+
+        str += " / ";
+        str += (
+            hrs < 10 ? '0' : '') + hrs + ':';
+        str += (
+            min < 10 ? '0' : '') + min + ':';
+        str += (
+            sec < 10 ? '0' : '') + sec;
```
Comment (jacobkeeler) [inline app/model/sdl/MediaModel.js:242]: Probably worth creating a helper function for this at this point:
```
    function time2str(time) {
        var position, str = '', hrs = 0, min = 0, sec = 0;
        hrs = parseInt(time / 3600), // hours
          min = parseInt(time / 60) % 60, // minutes
          sec = position % 60; // seconds

        str += " / ";
        str += (
            hrs < 10 ? '0' : '') + hrs + ':';
        str += (
            min < 10 ? '0' : '') + min + ':';
        str += (
            sec < 10 ? '0' : '') + sec;
      }
```

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: app/model/sdl/MediaModel.js [hunk 2]
```diff
@@ -226,20 +226,35 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
...
+      } else if(this.endTime !== 0) {
+        const timeRemaining = this.startTime - this.endTime - this.currTime;
+        hrs = parseInt(timeRemaining / 3600), // hours
+          min = parseInt(timeRemaining / 60) % 60, // minutes
+          sec = timeRemaining % 60; // seconds
+
+        str += " / (-";
```
Comment (jacobkeeler) [inline app/model/sdl/MediaModel.js:249]:
```suggestion
        str += " (-";
```
I don't think the `/` makes much sense here

File: app/model/sdl/MediaModel.js (hunk 1 above)
Comment (ValeriiMalkov) [reply app/model/sdl/MediaModel.js:239]: @jacobkeeler Please take a look changes in commit 89fa2416fa8f97edba1ae4555e9fefa8407735a6

File: app/model/sdl/MediaModel.js (hunk 2 above)
Comment (ValeriiMalkov) [reply app/model/sdl/MediaModel.js]: @jacobkeeler Please take a look changes in commit 89fa2416fa8f97edba1ae4555e9fefa8407735a6

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: app/model/sdl/MediaModel.js [hunk 3]
```diff
@@ -226,20 +226,25 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
         min < 10 ? '0' : '') + min + ':';
     str += (
         sec < 10 ? '0' : '') + sec;
```
[the remark this answers is not in the document]
Comment (ValeriiMalkov) [reply app/model/sdl/MediaModel.js]: @jacobkeeler Done in commit 2495b8c2bc31b9b5f020f185d9498942ed48367a

**Review by jacobkeeler — APPROVED** [review]

**Pull request merged**